fetch_demos_common/include/fetch_demos_common/perception_clustering.h

```diff
@@ -0,0 +1,109 @@
+#ifndef FETCH_DEMOS_COMMON_PERCEPTION_CLUSTERING_H
+#define FETCH_DEMOS_COMMON_PERCEPTION_CLUSTERING_H
+#include <iostream>
+#include <string>
+#include <vector>
+#include <math.h> 
+// PCL
```

Comment:
You should check out “include what you use” IWYU

By moving most of the headers into the `.cpp` file, and only including what you actually use in the header file, you can speed up compiling especially if someone later reuses your header file.

Reply:
Got it! Will check out the IWYU